### PR TITLE
Adding COMBLE-MIP tutorial

### PR DESCRIPTION
**Pull request by twjuliano**
Including notebook for tutorial, as well as supporting script that contains functions and data files needed to run tutorial.

### CODE REVIEW

**Comment by twjuliano** [comment]
@mgrover1 looks like it is barking at the `glob` dependency.

File: environment.yml [hunk 1]
```diff
@@ -17,6 +17,11 @@ dependencies:
   - jupyter_server
   - xwrf
   - geoviews
+  - netCDF4
+  - numpy
+  - netCDF4
+  - pathlib
+  - glob
```
Comment (mgrover1) [inline environment.yml:24]:
```suggestion
```

glob is included by default

**Review by mgrover1 — CHANGES_REQUESTED** [review]
one small fix

**Review by mgrover1 — APPROVED** [review]

**Comment by mgrover1** [comment]
this looks great @twjuliano - do you have access to cumulus?

**Comment by twjuliano** [comment]
> this looks great @twjuliano - do you have access to cumulus?

@mgrover1 thanks! yes, I have access to cumulus

**Comment by mgrover1** [comment]
Any chance you could move the files you added here to that data folder on cumulus? 

`/gpfs/wolf2/arm/atm124/world-shared/arm-summer-school-2024`

**Comment by mgrover1** [comment]
Mainly just thinking about the size of the gitlog with the large files

**Review by mgrover1 — APPROVED** [review]

**Comment by mgrover1** [comment]
@twjuliano thanks for making those changes! Good to merge?

**Comment by twjuliano** [comment]
> @twjuliano thanks for making those changes! Good to merge?

@mgrover1 I say go for it!

**Pull request merged**